modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitionsLabelRenderer.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+interface IProps {
+	url: string;
+	value: LocalizedValue<string>;
+}
+
+export default function ViewObjectDefinitionsLabelRenderer({
+	url,
+	value,
+}: IProps) {
+	return (
+		<div className="table-list-title">
+			<a href={url}>
```

Comment:
i'm waiting the answer of @gabrielwas if this would be bad for security reasons (pass the url direclty)